CONTRIBUTING.md

```diff
@@ -8,11 +8,35 @@ Community contributions rock and we are psyched you're reading this document!
 - Feature requests are welcome and go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/feature-requests)
 - Lint rule proposals go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals)
 
+## How can I start contributing?
+
+### I don't want to write code, can I still contribute?
+
+Sure!
+
+You can always open a [discussion](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals) with a proposal for a new lint rule or other enhancement as well contributing to any open discussions. Ensure that you provide a clear and concise title and description of the desired addition. For lint rules in particular, WDL examples and/or links to the relevant portion of the WDL spec are extremely useful.
+
+We also welcome bug reports. If you discover a flaw in our codebase, please review the list of open issues to ensure that it is not a duplicate. Please also attempt to debug the issue locally and ensure that it is not a configuration issue. Once you have done both, please file a new issue providing the relevant information in the issue. Please provide the exact steps to reproduce the problem, specific example(s) that demonstrate the steps, and the behavior you observe as well as the behavior you expected to observe. For terminal-based use cases, a copy and paste of the command and error log (please use markdown formatting appropriately) is preferred. For interactive use cases (such as the VSCode extension), screenshots and/or GIFs are welcome ways to provide additional information to maintainers.
+
+We also appreciate feedback on our documentation. Feel free to look over any of our `*.md` files and note any issues you find. You can also explore our lint rule documentation by [installing `sprocket`](https://stjude-rust-labs.github.io/sprocket/installation.html) and reading the output of `sprocket explain`. (n.b.: we hope to replace `sprocket explain` with a website where each rule will have a dedicated page, but that has not been realized yet)
+
+### Your first code contribution
```

Comment:
this should come before "I don't want to write code". It seems more important

Reply:
I think it's important to ensure that non-developers understand their contributions are welcome, which is part of the reason why I put it first. I think the developers can easily get to the next section, but the non-code information isn't irrelevant for them either.

Reply:
ok. That's good reasoning 👍